### ISSUE TITLE
Dune version

### ISSUE THREAD

**Issue by Ninjapouet** [OCamlPro/drom#135]
Is it possible to change the dune version in order to use cram tests ?

**Comment by lefessan** [comment]
You may add `dune = "X.Y.Z"` in the `[tools]` section of the `drom.toml`, it should work, no ?

**Comment by Ninjapouet** [comment]
No, it adds the required opam dependencies but the (lang dune 2.0) still remain.

**Comment by lefessan** [comment]
so, there should be a project-wide option to see the dune lang ?

**Comment by Ninjapouet** [comment]
I suppose so :)

**Comment by lefessan** [comment]
This issue should be solved by #136 (when merged): 
You can add something like this in your `drom.toml` file:
```
[fields]
dune-lang = "2.2"
```

**Comment by Ninjapouet** [comment]
I wonder if this field should not trigger a correct dune dependency ? Setting `dune-lang = "2.7"` doesn't prevent drom from generating the opam dependency ` "dune" {>= "2.6.0"}` which allows the solver to select the 2.6.2 version which would contradict the lang specification.

**Comment by lefessan** [comment]
Indeed, but more complex to implement :-)
